docker/app/entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+cd "${APP_DIR}"
+
+mkdir -p tmp/pids
+rm -f tmp/pids/server.pid
+
+bin/rails db:prepare
+bin/rails runner 'DevTools.delete_all_indices_and_reindex_all' || echo "Warning: reindex failed, continuing..."
+
+exec bin/dev
```

Comment:
Then it runs `bin/dev` to start the application, including the Rails server, JavaScript build system, and Sidekiq.